extension/chrome/settings/modules/my_key.ts

```diff
@@ -36,17 +36,17 @@ View.run(class MyKeyView extends View {
     super();
     const uncheckedUrlParams = Url.parse(['acctEmail', 'fingerprint', 'parentTabId']);
     this.acctEmail = Assert.urlParamRequire.string(uncheckedUrlParams, 'acctEmail');
-    this.fingerprint = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'fingerprint') || 'primary';
+    this.fingerprint = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'fingerprint');
```

Comment:
How do I find `whichever places call it`? Made it required and now I have a failing test: https://github.com/FlowCrypt/flowcrypt-browser/blob/22a49cd0869343c50c92cdef22343517c1ec7435/test/source/tests/settings.ts#L49

I'm not sure how to fix it properly, could you please help me?

Reply:
It was showing a `my_key.htm` page complain that it's missing fingerprint. Find the places that open that page:

Reply:
Here: https://github.com/FlowCrypt/flowcrypt-browser/blob/2a58e6322fe96b087d5d5caa0a408b2354fe93d4/extension/chrome/settings/index.ts#L34
`acctEmail` is optional, so I can't fetch the primary (first) key to get its fingerprint and pass it to `my_key` page. Any ideas? Can we somehow make the `acctEmail` required? It's obvious it may be missing because there's a check: https://github.com/FlowCrypt/flowcrypt-browser/blob/2a58e6322fe96b087d5d5caa0a408b2354fe93d4/extension/chrome/settings/index.ts#L241

Reply:
I made an issue for this #3112 but for this current issue the `!` will work.